examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>coal mining link is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
is this the correct way? [coal mining](../getting_started.ipynb)

Reply:
Nope, use `` {ref}`pymc:pymc_overview` ``. It might not work yet but will once the repevant PR in pymc updating the notebook is merged. As this is rebuilt with every commit it should not take too long for everything to work without further action on our side